src/System/Logging/Category.cs

```diff
@@ -26,5 +26,54 @@ public Category(string name)
 		/// Name
 		/// </summary>
 		public string Name { get; }
+
+
+		/// <summary>
+		/// Indicates whether the current object is equal to another object of the same type
+		/// </summary>
+		/// <param name="other">An object to compare with this object</param>
+		public bool Equals(Category other)
+		{
+			if (ReferenceEquals(null, other))
+			{
+				return false;
+			}
+
+			if (ReferenceEquals(this, other))
+			{
+				return true;
+			}
+
+			return string.Equals(Name, other.Name, StringComparison.InvariantCulture);
+		}
+
+
+		/// <summary>
+		/// Determines whether the specified <see cref="object" />, is equal to this instance
+		/// </summary>
+		/// <param name="obj">The <see cref="object" /> to compare with this instance</param>
+		public override bool Equals(object obj) => ReferenceEquals(this, obj) || obj is Category other && Equals(other);
```

Comment:
Could you please wrap in parantheses to improve readability?

Reply:
Fixed, thanks